src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -87,7 +87,7 @@ def validate_ase_create(cmd, namespace):
 
 
 def _validate_asp_sku(sku, app_service_environment, zone_redundant):
-    if zone_redundant and get_sku_tier(sku).upper() not in ['PREMIUMV2', 'PREMIUMV3', 'ISOLATEDV2']:
+    if zone_redundant and get_sku_tier(sku).upper() not in ['PREMIUMV2', 'PREMIUMV3', 'PREMIUMMV3', 'PREMIUM0V3', 'ISOLATEDV2']:
```

Comment:
```suggestion 
    supported_skus = ['PREMIUMV2', 'PREMIUMV3', 'PREMIUMMV3', 'PREMIUM0V3', 'ISOLATEDV2']
    if zone_redundant and get_sku_tier(sku).upper() not in supported_skus:
```
This line is too long and it's failing the style check. We could break it over two lines like so ^